.github/workflows/ci-test-custom-script.yml

```
@@ -257,7 +257,7 @@ jobs:
 
       - uses: browser-actions/setup-chrome@latest
         with:
-          chrome-version: stable
+          chrome-version: 129
       - run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:40: Double quote to prevent globbing and word splitting

(shellcheck)